README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
-4. Run `sudo vim /etc/hosts` (or use your editor of choice) and add a line to the bottom with the following contents and save: `127.0.0.1 localhost.simplereport.gov`
-5. Follow these steps from GitHub for [authenticating to the container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry).
-6. Run `yarn start` in the root of the repository to start the app
-7. Make your user a support admin by assigning yourself the `SR-DEV-ADMINS` group in Okta Preview.
-8. When the frontend and backend builds are complete, visit https://localhost.simplereport.gov/app
-
-You should see the support admin page. The application does not yet start with any seed data, so you'll need to add your own organization and facilities. You can create your first org at https://localhost.simplereport.gov/app/sign-up.
-
-#### Notes
-Make sure docker has access to enough resources on your local machine. You'll know it doesn't if you get an error from any of your containers similar to this:
-```
-"The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process."
-```
-
-How to update resources limits on your [Mac](https://docs.docker.com/desktop/mac/#resources), and [Windows](https://docs.docker.com/desktop/windows/#resources) machines.
-
-## Running outside of docker
-
-If you wish to run outside of docker compose (bare metal), you'll need to install all the dependencies locally.
-See more [info here](https://github.com/CDCgov/prime-simplereport/wiki/Running-outside-of-docker).
-
-## Roles
-
-The available user role types are `ADMIN`, `USER`, `ENTRY_ONLY`, `ALL_FACILITIES`, and `NO_ACCESS`. You can check `backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java` for a list of available roles
-
-- `ADMIN` - an organization admin with full access to their organization
-- `USER` - a site user the has access to everything in their organization but the gear icon
-- `ENTRY_ONLY` - a site user that only has access to the Conduct Test tab
-- `ALL_FACILITIES` - a site user that can access all facilities in their organization
-- `NO_ACCESS` - a member of an organization who has no permissions without possessing other roles. Every member of an org has this role, so it is used to list all users in an organization
-
-These roles are controlled via Okta groups.
-
-## Support admin
-
-You can make your user a support admin by assigning yourself the `SR-DEV-ADMINS` group in Okta Preview.
-
-Support admins can access the `/admin` paths and support admin APIs.
-
-## Database cleaning
```

Comment:
⚠️ Moved to https://github.com/CDCgov/prime-simplereport/wiki/Database-maintenance